<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameDisplay</class>
 <widget class="QMainWindow" name="GameDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
background-image: url(://fondo.png);
}

#ven_tag{
color: red;
}

#ven_random{
background-image: url(://ven_random.png);
}

#ven_timer1{
background-image: url(://ven_timer1.png);
}

#ven_timer2{
background-image: url(://ven_timer2.png);
}

#ven_double{
background-image: url(://ven_double.png);
}

#ven_random2{
background-image: url(://ven_random.png);
}

#ven_timer01{
background-image: url(://ven_timer1.png);
}

#ven_timer02{
background-image: url(://ven_timer2.png);
}

#ven_double2{
background-image: url(://ven_double.png);
}

#target01{
background-image: url(://back.png);
}
#target02{
background-image: url(://back.png);
}
#target03{
background-image: url(://back.png);
}
#target04{
background-image: url(://back.png);
}
#target05{
background-image: url(://back.png);
}
#target06{
background-image: url(://back.png);
}
#target07{
background-image: url(://back.png);
}
#target08{
background-image: url(://back.png);
}
#target09{
background-image: url(://back.png);
}
#target010{
background-image: url(://back.png);
}
#target011{
background-image: url(://back.png);
}
#target012{
background-image: url(://back.png);
}
#target013{
background-image: url(://back.png);
}
#target014{
background-image: url(://back.png);
}
#target015{
background-image: url(://back.png);
}
#target016{
background-image: url(://back.png);
}
#target017{
background-image: url(://back.png);
}
#target018{
background-image: url(://back.png);
}

#target019{
background-image: url(://back.png);
}

#target020{
background-image: url(://back.png);
}

#target021{
background-image: url(://back.png);
}

#target022{
background-image: url(://back.png);
}

#target023{
background-image: url(://back.png);
}

#target024{
background-image: url(://back.png);
}

#title{
background-image: url(://title.png);
}

#timer_game{
color: white;
}

#numbytes{
color: red;
}

#Points{
color: red;
}

#Points_2{
color: white;
qproperty-alignment: AlignCenter;
background: black;
border: 3px red;
border-radius: 7px
}

#Points_22{
color: white;
qproperty-alignment: AlignCenter;
background: black;
border: 3px red;
border-radius: 7px
}

#Points_3{
color: red;
}

#Points_4{
color:red;
}

#Points_5{
color:red;
}

#Points_6{
color:red;
}


#playertag1{
color: black;
qproperty-alignment: AlignCenter;
background: red;
border: 8px red;
border-radius: 7px
}

#playertag2{
color: black;
qproperty-alignment: AlignCenter;
background: red;
border: 8px red;
border-radius: 7px
}

#play1{
color: black;
qproperty-alignment: AlignCenter;
background: black;
border: 8px red;
border-radius: 7px
}

#play2{
color: black;
qproperty-alignment: AlignCenter;
background: black;
border: 8px red;
border-radius: 7px
}




</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>911</width>
      <height>691</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="target01">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target02">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target03">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target04">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target05">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target08">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>180</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target06">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target07">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>180</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target09">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target012">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>350</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target010">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>350</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target011">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>350</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target013">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target014">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>180</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target015">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>350</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target017">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>350</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target018">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>180</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target016">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>10</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target019">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>520</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target020">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>520</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target021">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>520</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target022">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>520</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target023">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>520</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="target024">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>520</y>
       <width>130</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>-10</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="timer_game">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>50</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>123</string>
    </property>
   </widget>
   <widget class="QLabel" name="Points">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>640</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>POINTS</string>
    </property>
   </widget>
   <widget class="QLabel" name="Points_2">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Points_3">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TIMER</string>
    </property>
   </widget>
   <widget class="QLabel" name="playertag1">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PLAYER </string>
    </property>
   </widget>
   <widget class="QLabel" name="Points_4">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>90</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PLAYER 1 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="ven_tag">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="playertag2">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>390</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PLAYER </string>
    </property>
   </widget>
   <widget class="QLabel" name="Points_5">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>390</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PLAYER  2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Points_6">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>330</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>POINTS</string>
    </property>
   </widget>
   <widget class="QLabel" name="Points_22">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>640</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="play2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>390</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_tag1">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>130</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="ven_double">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ven_timer1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ven_timer2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ven_random">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_tag2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>440</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="ven_random2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ven_timer01">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ven_timer02">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ven_double2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="play1">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>90</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="numbytes">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>640</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
